### PR TITLE
CIV-14239 Response section - LiP View Applications

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-14239

### Change description ###
Response section - LiP View Applications

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -3147,7 +3147,12 @@
         "JUDGE_HAS_MADE_ORDER_DOCUMENT": "Judges Directions Order.doc",
         "PAGE_TITLE": "View application",
         "READ_RESPONSE": "Read the response",
-        "RESPONSE": "Response from the other parties",
+        "RESPONSE": {
+          "DO_YOU_AGREE_WITH_APLICANT_REQUEST": "Do you agree that the court should make the order that the applicant has requested?",
```
Comment (paul-pearson) [inline src/main/modules/i18n/locales/cy.json:3151]: typo - should be APPLICANT

File: src/main/modules/i18n/locales/en.json [hunk 2]
```diff
@@ -3147,7 +3147,12 @@
         "JUDGE_HAS_MADE_ORDER_DOCUMENT": "Judges Directions Order.doc",
         "PAGE_TITLE": "View application",
         "READ_RESPONSE": "Read the response",
-        "RESPONSE": "Response from the other parties",
+        "RESPONSE": {
+          "DO_YOU_AGREE_WITH_APLICANT_REQUEST": "Do you agree that the court should make the order that the applicant has requested?",
```
Comment (paul-pearson) [inline src/main/modules/i18n/locales/en.json:3151]: typo - should be APPLICANT

File: src/main/services/features/generalApplication/viewApplication/addViewApplicationResponseRows.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import { t } from 'i18next';
+import { exhaustiveMatchingGuard } from 'services/genericService';
+import { fromCcdHearingType } from 'services/translation/generalApplication/ccdTranslation';
+
+export const buildResponseSummaries = (generalApplication: CCDApplication, lng: string): SummaryRow[] => {
+  const responses = generalApplication.respondentsResponses;
+  const response = (responses?.length > 0) ? responses[0].value : undefined;
+  
+  const gaRespondentDebtorOfferSection = (): SummaryRow[] =>
+    [row('PAGES.GENERAL_APPLICATION.VIEW_APPLICATION.RESPONSE.DO_YOU_AGREE_WITH_APLICANT_REQUEST',
```
Comment (paul-pearson) [inline src/main/services/features/generalApplication/viewApplication/addViewApplicationResponseRows.ts:17]: typo - should be APPLICANT

File: src/test/unit/services/features/generalApplication/viewApplication/addViewApplicationResponseRows.test.ts [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+      const application: Partial<CCDApplication> = {
+        gaRespondentDebtorOffer: {
+          respondentDebtorOffer: CcdGARespondentDebtorOfferOptionsGAspec.ACCEPT,
+          debtorObjections: 'I disagree',
+        },
+        respondentsResponses: [{ value : {}} as CcdGeneralApplicationRespondentResponse],
+      } satisfies Partial<CCDApplication>;
+  
+      expect(buildResponseSummaries(application as CCDApplication, 'en')).toStrictEqual([{
+        key: { text: 'PAGES.GENERAL_APPLICATION.VIEW_APPLICATION.RESPONSE.DO_YOU_AGREE_WITH_APLICANT_REQUEST'},
```
Comment (paul-pearson) [inline src/test/unit/services/features/generalApplication/viewApplication/addViewApplicationResponseRows.test.ts:25]: typo - should be APPLICANT

File: src/test/unit/services/features/generalApplication/viewApplication/addViewApplicationResponseRows.test.ts [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+              CcdSupportRequirement.SIGN_INTERPRETER,
+              CcdSupportRequirement.HEARING_LOOPS,
+              CcdSupportRequirement.DISABLED_ACCESS,
+              CcdSupportRequirement.OTHER_SUPPORT,
+            ]} as CcdGeneralApplicationHearingDetails,
+        }}],
+      };
+
+      expect(buildResponseSummaries(application as CCDApplication, 'en')).toStrictEqual([{
+        key: { text: 'PAGES.GENERAL_APPLICATION.VIEW_APPLICATION.RESPONSE.DO_YOU_AGREE_WITH_APLICANT_REQUEST'},
```
Comment (paul-pearson) [inline src/test/unit/services/features/generalApplication/viewApplication/addViewApplicationResponseRows.test.ts:131]: typo - should be APPLICANT

**Review by paul-pearson — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**